### DISCUSSION TITLE
Updating/work-with this template for GHA and Azd Devcontainer feature.

### DISCUSSION THREAD

**Discussion by zedy-wj** [rpothin/servicebus-csharp-function-dataverse#6] [Ideas]
We are updating this template to support new features in `Github Action` and `Azd Devcontainer`.

1. For the Github Action feature: We will remove two lines of code on container image and add GHA code after checkout step in the `.github/workflows/provision-deploy.yml` file. (Shown as below)
<img width="302" alt="image" src="https://github.com/Azure-Samples/openai-plugin-fastapi/assets/81678720/b78e7045-08c4-4e09-90cd-ba0b22f50fe7">

2. For the Azd Devcontainer feature: We will remove dockerfile in `.devcontainer` folder and add  the following code in `devcontainer.json` file,

``` 
"ghcr.io/azure/azure-dev/azd:latest": {}
``` 
and image also need to be add instead of build. Such as:
<img width="275" alt="image" src="https://github.com/Azure-Samples/openai-plugin-fastapi/assets/81678720/c95f0cbb-06f3-49a9-84b3-7c90df29e6a7">

@rajeshkamal5050 , @rpothin for notification.

**Comment by rpothin** [comment]
Hey @zedy-wj 

Thank you very much for your help keeping the template up to date.
It was in my todo list but not yet prioritized 😅.

Have a great day.
